docs/design/2022-03-10-write-bypass-tidb.md

```diff
@@ -0,0 +1,117 @@
+# Flink connector write bypass TiDB
+
+- Author(s): [yangxin](http://github.com/xuanyu66)
+- Tracking Issue: https://github.com/tidb-incubator/TiBigData/issues/163
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Motivation or Background](#motivation-or-background)
+* [Detailed Design](#detailed-design)
+* [Compatibility](#compatibility)
+* [Test Design](#test-design)
+    * [Functional Tests](#functional-tests)
+    * [Scenario Tests](#scenario-tests)
+    * [Compatibility Tests](#compatibility-tests)
+    * [Benchmark Tests](#benchmark-tests)
+* [Impacts & Risks](#impacts--risks)
+* [Investigation & Alternatives](#investigation--alternatives)
+* [References](#references)
+
+## Introduction
+
+This article introduces an optimized solution for writing Flink connector to TiDB cluster, by bypassing TiDB server and writing TiKV directly.
+
+## Motivation or Background
+
+Currently, Fink connector uses JdbcDynamicTableSink to write data through TiDB server.
+
+However, this approach leads to the following problems.
+- Lack of atomicity, if the Flink task fails and exists during the writing process, some data will be written successfully while the others not.
+- Lack of isolation, other transactions can see part of the written data during the writing process.
```

Comment:
Gives me a feeling TiDB doesn't support transaction, which is not the fact.

Reply:
Give a description that JdbcDynamicTableSink doesn't provide transaction semantics, not TiDB